Bump mono to get archives built with Xcode 11.1.

rolfbjarne opened the pull request:
New commits in mono/mono:

* mono/mono@b601371d5f0 Update MERP event type to MonoAppCrash
* mono/mono@6184ff007b2 [2019-08][ci] Use Xcode11.1 and 11.2beta2 for XI/XM Mono SDK builds (#17324)
* mono/mono@8969f2cc99b [2019-08] [merp] Include any managed methods in the 'unmanaged_frames' portion … (#17316)
* mono/mono@30094401081 [2019-08][merp] Don't install SIGTERM handler in EnableMicrosoftTelemetry (#17308)

Diff: https://github.com/mono/mono/compare/df5e13f95df7a2d11d86904e74b1bd8950c9d43b..b601371d5f0b853b133608be25a18ea37f7628ed

monojenkins commented:
Build failure
✅ [Build succeeded](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11594//console)
✅ [API Diff (from stable)](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11594//API_20diff_20_28from_20stable_29)
🔥 [Failed to compare API and create generator diff](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11594//console) 🔥
&nbsp;&nbsp;&nbsp;&nbsp;Failed to update apidiff references
&nbsp;&nbsp;&nbsp;&nbsp;Search for `Comparing API & creating generator diff` in the log to view the complete log.
🔥 [Test run failed](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11594//Test_20Report) 🔥

rolfbjarne commented:
build

monojenkins commented:
Build failure
✅ [Build succeeded](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11602//console)
✅ [API Diff (from stable)](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11602//API_20diff_20_28from_20stable_29)
❎ Skipped API comparison because the PR has the label 'skip-api-comparison'
🔥 [Test run failed](https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11602//Test_20Report) 🔥

# Test results

<details>
<summary>9 tests failed, 142 tests passed.</summary>

## Failed tests

 * introspection/Mac Modern/Debug: Failed (Test run failed.)
 * xammac tests/Mac Modern/Release: Failed (Test run failed.)
 * monotouch-test/iOS Unified 64-bits - simulator/Debug: Crashed
 * monotouch-test/iOS Unified 64-bits - simulator/Debug (static registrar): Crashed
 * monotouch-test/tvOS - simulator/Debug: Failed
 * monotouch-test/tvOS - simulator/Debug (static registrar): Failed
 * monotouch-test/watchOS 32-bits - simulator/Debug: Crashed
 * mmptest/macOS/Debug: BuildFailure
 * MTouch tests/NUnit: Failed (Execution failed with exit code 3)
</details>

rolfbjarne commented:
Test failures are unrelated

* introspection/Mac: Fixed with #7239.
* mmptest/macOS/Debug: https://github.com/xamarin/maccore/issues/2031
* xammac tests/Mac Modern/Release: https://github.com/xamarin/maccore/issues/2032
* monotouch-test/iOS Unified 64-bits - simulator/Debug: https://github.com/xamarin/maccore/issues/2030
* monotouch-test/iOS Unified 64-bits - simulator/Debug (static registrar): https://github.com/xamarin/maccore/issues/2030
* monotouch-test/tvOS - simulator/Debug: https://github.com/xamarin/maccore/issues/2030
* monotouch-test/tvOS - simulator/Debug (static registrar): https://github.com/xamarin/maccore/issues/2030
* monotouch-test/watchOS 32-bits - simulator/Debug: https://github.com/xamarin/maccore/issues/2030
* MTouch tests/NUnit: Fixed with #7241.
